src/scrabble.js

```diff
@@ -1,5 +1,57 @@
-function scrabble() {
+const values = require('./letter-values.js') // fetch the values table
 
+function scrabble(word) {
+  let points = 0
+  if (word != null) {
+    // safeguarding against no value
+    const character = word.split('')
+    // split the given word into individual characters in an array
+    if (word.includes('{', '}')) {
+      // if word contains brackets
+      const firstBracket = word.indexOf('{') // creates variable that's the index of the first bracket location
```

Comment:
This variable is interesting because it is very specific and used only for one purpose.  I'm not sure, but could this have been made into one variable for both brackets? (overall thumbs up)

Reply:
It likely could have been, this was just my first implementation and I really should have gone back over it to see if it could be refined, thanks for the feedback.